tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -52,6 +52,7 @@ quota.consumer.bytes.per.second.overrides={{ quota_config.quota_consumer_bytes_p
 
 security.inter.broker.protocol={{ interbroker_security_protocol }}
 
+ssl.client.auth=required
```

Comment:
Actually - I retract that. Pushing this into the authoriser conditional fixed one of the tests! I've pushed a change to flavio's branch